server/commands/src/impls.rs

```diff
@@ -28,7 +33,11 @@ pub fn tp_1(ctx: &mut CommandCtx, destination: EntitySelector) -> anyhow::Result
             teleport_entity_to_pos(&mut ctx.world, ctx.sender, pos);
         }
 
-        Ok(())
+        Ok(Some(format!(
+            "Teleported {0} to {1}",
+            ctx.world.try_get::<Name>(ctx.sender).unwrap().0.to_string(),
```

Comment:
Instead of `try_get().unwrap()`, it's possible to call `get()` to unwrap the result automatically.

Reply:
Think that's done :)